### PR TITLE
Updating README

### PR DESCRIPTION
**Pull request by zyronix**
#### Pull Request (PR) description
Updating the README so it applies to the current state of the module.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,183 +6,111 @@
...
+
+## Usage
+### Server
+#### Basic usage
+The following code will install check_mk and create one 'omd site'. Once this is done you should be able to access it by visiting:
+
+```http://<IP of the machine>/monitoring/```
+
+Debian 9:
+```
```
Comment (alexjfisher) [inline README.md]: You could mark the example as puppet code.  ie use...
```
```puppet
```

File: README.md [hunk 2]
```diff
@@ -6,183 +6,111 @@
...
+  filestore => 'http://<url of the check_mk server>/monitoring/check_mk/agents/',
+  package   => 'check-mk-agent-1.5.0p19-1.noarch.rpm',
+}
+```
+
+#### Securing agent
+The agent has the ability to implement a whitelist of check_mk servers to limited access only to those servers. To increase the security of the check_mk agent you can implement the following:
+  * Encrypt communication with a secret. See [the check_mk website](https://checkmk.com/cms_agent_linux.html#encryption) for more information.
+  * Implement whitelisting on the agent.
+  * Implement a strict incoming firewall which only allows access on port 6556 from the check_mk server. The firewall can be configured using the puppetlabs/firewall module.
```
Comment (alexjfisher) [inline README.md]: or crayfishx/firewalld??

Comment (alexjfisher) [reply README.md]: (soon to become `puppet/firewalld` BTW) :)

File: README.md (hunk 1 above)
Comment (zyronix) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (zyronix) [reply README.md]: Happy to change it once it is part of voxpupuli :) but the reference is just a suggestion. I'll leave it for now the way it is written;

**Pull request merged**